packages/generate-prism-languages/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "generate-prism-languages",
+  "private": true,
+  "scripts": {
+    "start": "ts-node ./index.ts"
+  },
+  "peerDependencies": {
+    "react": ">=16.0.0"
+  },
+  "devDependencies": {
+    "@types/lodash.flowright": "^3.5.7",
+    "@types/node": "^18.15.11",
+    "@types/prismjs": "^1.26.0",
+    "@types/uglify-js": "^3.17.1",
+    "picocolors": "^1.0.0",
+    "prismjs": "*",
+    "ts-node": "^10.9.1",
+    "tslib": "^2.5.0",
+    "typescript": "*",
+    "uglify-js": "^3.17.4"
+  },
+  "dependencies": {
+    "lodash.flowright": "^3.5.0"
```

Comment:
Any reason to not just move this into devDeps?

Reply:
Haha no, I must have not capitalized the `-D` when running `pnpm add`